server/api.js

```diff
@@ -84,4 +85,88 @@ router.get("/quizes",(req, res)=>{
 	});
 });
 
+router.get("/quizes/search",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("quizes");
+		const quiz = { quiz_name:req.query.quiz_name };
+		collection.findOne(quiz,(err, questions)=>{
+			if(err){
+				res.status(500).send(err);
+			}else{
+				res.status(200).send(questions);
+			}
+		});
+	});
+});
+
+router.get("/results/search",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("results");
+		const quiz = { name:req.query.name };
+		collection.find(quiz).toArray((err, results)=>{
+			if(err){
+				res.status(500).send(err);
+			}else{
+				res.status(200).send(results);
+			}
+		});
+	});
+});
+router.get("/results",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("results");
+		collection.find().toArray((err, results)=>{
```

Comment:
This is good for now, but if it's ultimately going to hand back all results for all quizzes we'll want to add some sort of limit/pagination to it so the database/server doesn't get overloaded.

(Definitely something that can be added in the next iteration though.)

Reply:
Yes I agree with you, I'll skip it for now and do it in the next iteration.